test/integration/fixtures/multiple-runs/multiple-runs-different-output.fixture.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const Mocha = require('../../../../lib/mocha');
+
+const mocha = new Mocha({ reporter: 'json' });
+if (process.argv.indexOf('--no-clean-references') >= 0) {
```

Comment:
Is there precedent for sending these custom arguments elsewhere?  It may have unintended consequences.  a more foolproof could be to modify `env` in the options object (third parameter to `invokeMocha`)

is the reason for these just that you can reuse the fixture in several different ways?  if that's the case, I'd probably prefer just copying the fixtures into separate files, each with their own behavior.  if you need to reuse some code, put that in a common module and `require` it from the fixtures

Reply:
> is the reason for these just that you can reuse the fixture in several different ways

Yes, they are. Will make 3 fixtures for them 👩‍👧‍👧